cfme/tests/configure/test_workers.py

```diff
@@ -27,3 +44,45 @@ def test_restart_workers(appliance):
     wait_for(lambda: len(pids) == len(worker.get_all_worker_pids()),
              fail_func=worker.parent.reload_workers_page, num_sec=1800, delay=10,
              message="Wait for all original workers are back online")
+
+
+@pytest.mark.tier(0)
```

Comment:
@izapolsk, I initially thought it was tier(0) based on the fact that the key functionality of this page is broken. A page that lets you make changes should be reflective of the changes that you make. Though, since the changes are actually made, just not communicated to you, I think that perhaps a tier(2) would be more appropriate.